Changelog.md

```diff
@@ -1,6 +1,27 @@
 ## libsdformat 13.X
 
-### libsdformat 13.3.0 (2022-12-12)
+### libsdformat 13.3.0 (2022-02-07)
+
+1. Use `File.exist?` for Ruby 3.2 compatibility
+    * [Pull request #1216](https://github.com/gazebosim/sdformat/pull/1216)
+
+1. Make ThrowOrPrintError a free internal function
+    * [Pull request #1221](https://github.com/gazebosim/sdformat/pull/1221)
+
+1. Make throwOrPrintErrors a member of sdf::Error
```

Comment:
nit: These two can be combined

Reply:
combined, and left the PR description to be `Make ThrowOrPrintError a free internal function`